nuxeo-services/nuxeo-platform-test/src/main/java/org/nuxeo/ecm/platform/test/PlatformFeature.java

```diff
@@ -31,11 +31,8 @@
 @Deploy("org.nuxeo.ecm.platform.api")
 @Deploy("org.nuxeo.ecm.platform.content.template")
 @Deploy("org.nuxeo.ecm.platform.dublincore")
-@Deploy("org.nuxeo.ecm.platform.usermanager.api")
-@Deploy("org.nuxeo.ecm.platform.usermanager")
 @Deploy("org.nuxeo.ecm.platform.query.api")
-@Deploy("org.nuxeo.ecm.platform.test:test-usermanagerimpl/directory-config.xml")
-@Features({ CoreFeature.class, ClientLoginFeature.class, DirectoryFeature.class })
+@Features({ CoreFeature.class, ClientLoginFeature.class, DirectoryFeature.class, UserManagerFeature.class })
```

Comment:
Cant' you remove the DirectoryFeature as it is included by the UserManagerFeature?

Reply:
I can thanks to transitive dependency, but I didn't know if it was what we want. For instance `DirectoryFeature` depends on `CoreFeature` to have access to backend type under test afaik. We might want to remove it one day and in such case will we want to hunt every places to upgrade? I tried to remove unneeded features while we don't require it explicitly, for instance `NuxeoLoginFeature` depends on `UserManager` but don't care about directory implementation detail. Wdyt?

Reply:
Good point, and it's also better to keep it explicit, not needing to introspect the features' hierarchy to understand the transitive dependencies.